cosmos/airflow/graph.py

```diff
@@ -125,6 +126,18 @@ def create_task_metadata(
             task_id = f"{node.name}_run"
             if use_task_group is True:
                 task_id = "run"
+        if node.resource_type == DbtResourceType.SOURCE:
+            task_full_name = node.unique_id[len("source.") :]
+            task_id = f"{task_full_name}_source"
+            args["select"] = f"source:{node.unique_id[len('source.'):]}"
+            args["models"] = None
```

Comment:
@tatiana I have a feeling that this lines can be written somewhere else. 
My dbt testing showed that 
`dbt source freshness --models [any] ` fails, while using the --select works. I ended up manually setting as None the models argument.